File: Frontend/src/app/DTOs/GeneralQuestion/GeneralQuestionQuestionDTO.ts
import { QuestionType } from "src/app/Enums/QuestionType";
import { ChallengeQuestionAnswerDTO } from "../Challenge/ChallengeQuestionAnswerDTO";

export interface GeneralQuestionQuestionDTO {
    question: string;
    description: string | null;
    questionType: QuestionType;
    answerSetOne: ChallengeQuestionAnswerDTO[] | null;
    answerSetTwo: ChallengeQuestionAnswerDTO[] | null;
}

File: Backend/DTOs/GeneralQuestion/GeneralQuestionQuestionDTO.cs
namespace Learnz.DTOs;
public class GeneralQuestionQuestionDTO
{
    public string Question { get; set; }
    public string? Description { get; set; }
    public QuestionType QuestionType { get; set; }
    public List<ChallengeQuestionAnswerDTO>? AnswerSetOne { get; set; }
    public List<ChallengeQuestionAnswerDTO>? AnswerSetTwo { get; set; }
}